src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -79,7 +82,12 @@ const UserProfile = ({
     if (isAnonymous) {
       return (
         <View style={styles.profile}>
-          <Image source={AnonymousProfile} width={32} height={32} style={styles.image} />
+          <Image
+            source={AnonymousProfile}
+            width={32}
+            height={32}
+            style={[styles.image, styles.anonImageBg]}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Image` component from `react-native` does not accept `width` and `height` as props. Instead, these should be specified in the style prop.
```diff
-          <Image
-            source={AnonymousProfile}
-            width={32}
-            height={32}
-            style={[styles.image, styles.anonImageBg]}
-          />
+          <Image
+            source={AnonymousProfile}
+            style={[styles.image, styles.anonImageBg, {width: 32, height: 32}]}
+          />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->